docs/dev/stream/state/state.md

```diff
@@ -400,6 +400,17 @@ This option is not applicable for the incremental checkpointing in the RocksDB s
 - For existing jobs, this cleanup strategy can be activated or deactivated anytime in `StateTtlConfig`, 
 e.g. after restart from savepoint.
 
+#### Cleanup in background
+
+Besides cleanup in full snapshot, you also can activate the cleanup in background. The option will adapt
+all the cleanup strategies with default values. Users who do not need to think about details or used backend.
+Currently, Flink provide two kind of cleanup strategies in background and the default config value list below:
+
+* Incremental cleanup(cleanup size: 10, run cleanup for every record: true)
+* Cleanup during RocksDB compaction(query time after how many entries: 1000)
+
+If users who want to get more fine-grained control about the special cleanup in background can use it directly.
+
```

Comment:
We also need a code example here:
```
Besides cleanup in full snapshot, you can also activate the cleanup in background. This option will activate
a strategy configured by default if the background cleanup is supported for the used backend. 

This feature can be activated in `StateTtlConfig`:
<<<<<< ... code example is needed like in other chapters .... >>>>>

For more fine-grained control over some special cleanup in background, you can configure it separately as described below.
```

This should go to the respective chapters where the parameter values are discussed:
```
 * Incremental cleanup(cleanup size: 10, run cleanup for every record: true)
 * Cleanup during RocksDB compaction(query time after how many entries: 1000)
```